2016/july/raising_money_for_your_(first)_python_workshop_with_corporate_sponsorships.md

```diff
@@ -0,0 +1,74 @@
+##Raising money for your (First) Python workshop with corporate sponsorships
+####By Trey Hunner & Lorena Mesa
+
+*Myth: Companies aren't interested in your events.* 
```

Comment:
A couple things I'd thought of in regards to structure of the post:
- Each bit of information is a direct response to a perceived problem/question (sort of an issue/fix approach)
- I think it might be good to have an intro followed by headers for each sub-section (maybe the headers should be topics instead of questions?)